<ui version="4.0" >
 <class>ModifUtilDialog</class>
 <widget class="QWidget" name="ModifUtilDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Modification des comptes</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QTableView" name="tableViewUtil" />
     </item>
     <item row="1" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>&amp;Enregistrer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3" >
         <property name="text" >
          <string>Eff&amp;acer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>&amp;Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
